app/src/main/java/org/xwiki/android/sync/activities/SettingServerIpViewFlipper.kt

```diff
@@ -68,8 +70,8 @@ class SettingServerIpViewFlipper(activity: AuthenticatorActivity, contentRootVie
         val serverAddress = serverEditText.text.toString()
 
         if (TextUtils.isEmpty(serverAddress)) {
-            serverEditText.error = mContext.getString(R.string.error_field_required)
             serverEditText.requestFocus()
+            serverEditText.error = mContext.getString(R.string.error_field_required)
```

Comment:
Are you sure that it is required change (I mean replace of this line)

Reply:
No, I'll revert it.

Reply:
Updated.